dataprofiler/data_readers/data_utils.py

```diff
@@ -299,13 +327,18 @@ def read_csv_df(
 
     # if the buffer was wrapped, detach it before returning
     if is_buf_wrapped:
+        assert isinstance(file_path, TextIOWrapper)
```

Comment:
Should the boolean `is_buf_wrapped` be removed and we can just check `if isinstance(file_path, TextIOWrapper)`?

Reply:
That should work and would simplify the static typing.